packages/account-management/src/routes/index.ts

```diff
@@ -1,9 +1,12 @@
 import { RouteMiddleware } from '../types/application';
 
+const unAuthenticatedPages: string[] = ['/register'];
+
 export const indexPage: RouteMiddleware = (context) => {
   const bundle = context.routes.url('assets-bundles');
 
-  if (!context.isAuthenticated()) {
+  const isUnauthenticatedPage: boolean = !!context.req.url && unAuthenticatedPages.includes(context.req.url);
```

Comment:
Is there a situation where `req` wouldn't have a `url` here? Do we need to check that it exists?

Reply:
In practice for us, probably not, but it's typed as optional with a comment that I think explains why:

```
        /**
         * Only valid for request obtained from http.Server.
         */
        url?: string;
```

Reply:
Presumably there's a way of getting hold of the request that doesn't populate it

Reply:
Mental. Very philosophical: "What is a request, without a URL?"

We're good here though :+1: